package/kernel/mt76/Makefile

```diff
@@ -218,6 +218,12 @@ define KernelPackage/mt7663u
   AUTOLOAD:=$(call AutoProbe,mt7663u)
 endef
 
+define KernelPackage/mt7915-firmware
```

Comment:
I think you should change the default packages for the boards which need the new `mt7915-firmware` already in the same commit where you introduce it. Same for the one which adds `mt7622-firmware`

This will keep OpenWrt bisectable. When someone  wants to use `git bisect` on OpenWrt and is getting between the commit which creates `mt7915-firmware` and the one which adapts the default packages for the boards, some firmware files will be missing in the image.

Reply:
Thanks, I reorganized them and applied the changes to the recently added devices.